src/components/Welcome.tsx

```diff
@@ -41,7 +41,7 @@ const Welcome = () => {
           transition={{ delay: 0.8, duration: 0.5 }}
           className="w-full h-[35%] flex justify-around items-center flex-col"
         >
-          <p className="text-2xl font-semibold">Welcome to MarketPlace</p>
+          <p className="text-2xl font-semibold">Welcome to maarketPlaace</p>
```

Comment:
<p className="text-2xl font-semibold">Welcome to maarketplaace</p>
you can make that p small on maarketplaace to be small later